### ISSUE TITLE
[Other] No VA.gov Experience Standard for the issue found. (00.00.4)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#54708]
#### VFS team name
GovCIO-VFEP

#### VFS product name
GI Bill Comparison Tool

#### VFS feature name
Institution Ratings System

#### Point of Contact/Reviewers
Brian DeConinck - @briandeconinck - Accessibility

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
No VA.gov Experience Standard for the issue found.

### Issue Details
[This was encountered while reviewing the institution ratings, but has been confirmed to be an existing problem in production.]

When comparing two or more institutions, after checking each box keyboard-only users must [Tab] through every other item in the search results to get to the "Compare institutions" drawer and the "Compare" button.

When searching by institution name and at most 10 results appear at a time, that means keyboard-only may need to press [Tab] maybe 25 times to reach the "Compare" button (depending on how many pages in the pagination and how many schools have cautionary warnings).
- For sighted keyboard-only users, this can be a frustrating experience. They can see the "Compare institutions" drawer but they don't have any clear indication of where it sits in the DOM and how many times they'll have to press [Tab].
- For screen reader users, the pattern is even more frustrating. The screen reader announces that an institution has been added for comparison, but without the visual affordance of the drawer at the bottom of the page there's no indication of what they're supposed to do next to actually compare the institutions. Some users may press [Tab] enough times to discover the drawer and the compare button, others may abandon it entirely.

When searching by location, a large urban area might have hundreds of results         with no pagination. Between the results list and the map markers, a keyboard-only user may need to press [Tab] thousands of times to reach the compare button. For both sighted keyboard-only users and screen reader users, this is not a task that can be reasonably be accomplished.

### Link, screenshot or steps to recreate
Steps to reproduce:
1. Navigate to /gi-bill-comparison-tool/, type a search term for "Search by name" and press [Enter].
2. Press [Tab] to navigate through the search results.
3. Choose two institutions to compare, pressing [Enter] to check the compare checkbox for each institution.
4. Press [Tab] until you reach the Compare button.
5. Repeat when searching by location.

### VA.gov Experience Standard
[Category Number 00, Issue Number 00](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Platform Recommendation
I don't have a recommendation here, except that the design choices that resulted in this layout should be revisited and revised. There *may* be ways to make this design accessible using spontaneous skip links or custom keyboard shortcuts, but those may be overengineering the page when a simpler alternative may be more desirable.

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.
